onadata/apps/api/tests/viewsets/test_dataview_viewset.py

```diff
@@ -86,7 +87,7 @@ def test_filter_to_field_lookup(self):
 
     def test_get_field_lookup(self):
         self.assertEqual(
-            get_field_lookup("q1", "="), "json__q1__iexact"
```

Comment:
Why are we removing the case insensitive search ?

Reply:
seems there was no need for this change, reverted!